app/models/pacbio/request_factory.rb

```diff
@@ -3,6 +3,115 @@
 module Pacbio
   # RequestFactory
   class RequestFactory
```

Comment:
Could add in the module level comments from the old `Pipelines::Requestor::...` etc 
e.g.
```
    # Factory - behaviour for pipeline requests factory
    # It will take an array of request parameters and build an
    # array of request and associated samples (if they dont exist)
    # Also manages validation and saving
```